sources/packages/backend/apps/api/src/route-controllers/models/common.dto.ts

```diff
@@ -111,3 +115,15 @@ export class FileCreateAPIOutDTO {
   size: number;
   mimetype: string;
 }
+
+/**
+ * Represents an error considered not critical for
+ * an offering or application withdrawal
+ * and provides an user-friendly message
+ * and a type that uniquely identifies this warning
+ * or info.
+ */
+export class ValidationResultAPIOutDTO {
+  typeCode: OfferingValidationWarnings | OfferingValidationInfos | string;
```

Comment:
Just wanted to check, like `OfferingValidationWarnings | OfferingValidationInfo`, can't we create an enum for bulk withdrawal? Because, since now we introduced `typeCode` as `string`, the first two types are compromised right?

Reply:
you are right @ann-aot , I will change it